src/util/tile_request_cache.js

```diff
@@ -17,6 +17,8 @@ export type ResponseOptions = {
     headers: window.Headers
 };
 
+const cacheRequest = window.caches ? window.caches.open(CACHE_NAME) : null;
```

Comment:
@ansis @kkaefer Is there a chance that these access or method will throw an exception? I'm not seeing any documented cases in the MDN docs, and have tested in Sfari, Firefox and Chrome with `localhost`, `127.0.0.1`, and  over `http://`. 
This should probably be tested with Windows browsers and in incognito modes as well.

Reply:
These functions generally shouldn't throw exceptions because they return promises. We use `Promise#catch` in some situations to cover the failure case.